src/Raven.Client/Documents/Subscriptions/DocumentSubscriptions.cs

```diff
@@ -457,5 +457,31 @@ public Task DisableAsync(string name, string database = null, CancellationToken
             var operation = new ToggleOngoingTaskStateOperation(name, OngoingTaskType.Subscription, disable: true);
             return _store.Maintenance.ForDatabase(database ?? _store.Database).SendAsync(operation, token);
         }
+
+        /// <summary>
+        /// Update a data subscription in a database.
+        /// </summary>
+        public string Update(SubscriptionCreationOptions options, string database = null, long? id = null)
```

Comment:
introduce `SubscriptionUpdateOptions` that inherit from `SubscriptionCreationOptions`. It will allow us to extend them in the future if needed. Maybe they will not be 1 to 1 with creation ones.

Reply:
We can move then the Id to the options and maybe do some validation client side.